tests/playwright-test/test-step.spec.ts

```diff
@@ -1468,44 +719,21 @@ test('should step w/ box', async ({ runInlineTest }) => {
   }, { reporter: '' });
 
   expect(result.exitCode).toBe(1);
-  const objects = result.outputLines.map(line => JSON.parse(line));
-  expect(objects).toEqual([
-    {
-      category: 'hook',
-      title: 'Before Hooks',
-    },
-    {
-      title: 'boxed step',
-      category: 'test.step',
-      error: expect.not.stringMatching(/a.test.ts:[^8]/),
-      location: { file: 'a.test.ts', line: 8, column: 21 },
-      steps: [{
-        title: 'expect.toBe',
-        category: 'expect',
-        error: expect.stringContaining('expect(received).toBe(expected)'),
-        location: { file: 'a.test.ts', column: 29, line: 5 }
-      }],
-    },
-    {
-      category: 'hook',
-      title: 'After Hooks',
-    },
-    {
-      category: 'hook',
-      title: 'Worker Cleanup',
-    },
-    {
-      error: {
-        message: expect.stringContaining('expect(received).toBe(expected)'),
-        stack: expect.not.stringMatching(/a.test.ts:[^8]/),
-      }
-    }
-  ]);
+  expect(stripAnsi(result.output)).toBe(`
+hook      |Before Hooks
+test.step |boxed step @ a.test.ts:8
+test.step |↪ error: Error: expect(received).toBe(expected) // Object.is equality
+expect    |  expect.toBe @ a.test.ts:5
+expect    |  ↪ error: Error: expect(received).toBe(expected) // Object.is equality
+hook      |After Hooks
+hook      |Worker Cleanup
+          |Error: expect(received).toBe(expected) // Object.is equality
+`);
 });
 
 test('should soft step w/ box', async ({ runInlineTest }) => {
```

Comment:
Let's print error locations in `should soft step w/ box`, `should step w/ box` and `should step w/o box`.